### ISSUE TITLE
Error in project upon installation Error: Cannot find module './nix/nix'

### ISSUE THREAD

**Issue by shikhag2018** [carbon-app/carbon#1438]
**Describe the bug**
Error: Cannot find module './nix/nix'  on installing project and running yarn dev

**To Reproduce**
Install project

Run 
yarn install
yarn dev
open browser http://localhost:3000

**Expected behavior**
Should work fine

**Screenshots**
![Screenshot from 2022-12-15 21-41-18](https://user-images.githubusercontent.com/40258568/207911214-c4ed1248-d6d0-412c-9982-c145acf3e807.png)

**Info (please complete the following information):**

- OS [Linux 18.04]

Was able to resolve it by commenting these lines of code in lib/constants.js
```
 {
   name: 'Nix',
    mode: 'nix',
   highlight: true,
  },

  {
    name: 'Stan',
  mode: 'stan',
  highlight: true,
  },

```

**Comment by umi0410** [comment]
I also suffered this issue and the workaround worked. Thank you.

**Comment by shikhag2018** [comment]
@umi0410 I am glad it Worked for you

**Comment by agilityhawk** [comment]
Seeing the same issue. 

Propose fix: 
Should we add a check in [components/Carbon.js at Line #397](https://github.com/carbon-app/carbon/blob/main/components/Carbon.js#L397) to ignore the missing file, write a log and continue without errors?

**Comment by skevprog** [comment]
Created a PR for this https://github.com/carbon-app/carbon/pull/1448.

**Comment by pierow2k** [comment]
I've come across the same error upon updating my existing Carbon setup. With the generic output from `start`, it wasn't immediately clear what was triggering the problem. Running in `dev` mode, however, made the error evident, which brought me to this issue. After implementing the fix from #1448, it all appears to be working smoothly.

Thank you @skevprog!

**Comment by Musicminion** [comment]
Oh, same issue, PR for this https://github.com/carbon-app/carbon/pull/1448 works well, but no one merges?
